recipes/lcensemble/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "lcensemble" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lcensemble-{{ version }}.tar.gz
+  sha256: ba00723c53edf32efe96814ccd07b704dc52d44f5137c583f335f7348f6da3de
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - hyperopt >=0.2.7
+    - matplotlib-base
+    - numpy >=1.19.2
```

Comment:
This is different from upstream. Is that correct?

Reply:
Hello,
Thank you for your message and I confirm that it is correct.